h5p-bildetema/language/nb.json

```diff
@@ -4,192 +4,192 @@
       "label": "Oversettelser",
       "fields": [
         {
-          "label": "Språkkode",
+          "label": "Language code",
```

Comment:
Should it not be still språkkode?

Reply:
We can keep "Språkkode", but keeping this field in English makes it easier to find the field in all language files by simply searching. Since all other field labels are changed to English, it's good practice to also change this.